connectors/google/google-sheets/src/main/java/io/camunda/connector/gsheets/model/request/GoogleSheetsRequest.java

```diff
@@ -15,7 +16,11 @@
 
 public class GoogleSheetsRequest extends GoogleBaseRequest {
 
-  @Key @Valid @NotNull private Input operation;
+  @NestedProperties(addNestedPath = false)
+  @Key
```

Comment:
Same as above, we should be able to remove the annotation `@Key` right?

Reply:
This one seems to be redundant from the beginning. I'll remove and retest it.

Reply:
Removed and tested.